backend/app/core/authz/actions.py

```diff
@@ -15,6 +15,8 @@ class AuthorizationAction(IntEnum):
     GLOBAL__CREATE_DATASET = 103
     GLOBAL__REQUEST_DATAPRODUCT_ACCESS = 104
     GLOBAL__REQUEST_DATASET_ACCESS = 105
+    GLOBAL__DELETE_USER = 106
+    GLOBAL__CREATE_USER = 107
```

Comment:
Dumb remark, but can we still swap the order while the PR is not merged? Usually we've put the DELETE after the corresponding CREATE action.

```suggestion
    GLOBAL__CREATE_USER = 106
    GLOBAL__DELETE_USER = 107
```